frontend/src/containers/data-tool/content/details/index.tsx

```diff
@@ -0,0 +1,40 @@
+import DashboardTable from '@/components/dashboard-table';
+import { Button } from '@/components/ui/button';
+import { useSyncDataToolContentSettings } from '@/containers/data-tool/sync-settings';
+
+import { mockedData } from './mocked-data';
+
+const DataToolDetails: React.FC = () => {
+  const [settings, setSettings] = useSyncDataToolContentSettings();
+
+  const handleOnCloseClick = () => {
+    setSettings({ ...settings, details: false });
```

Comment:
You can  do `setSettings((prevSettings) => ({ ...prevSettings, details: false }));` so you avoid calling `settings` in the component and unnecessary renders (every time `settings` changes, the component will render)

Reply:
Done. I followed the pattern you used for the map.